docsearch/hbs-helpers/versionless.js

```diff
@@ -1,3 +1,3 @@
-const versionless = (components) => components.filter(({ name, latest }) => name !== 'home' && name !== 'tutorials' && latest === 'master')
+const versionless = (components) => components.filter(({ name, latest }) => name !== 'home' && name !== 'tutorials' && (latest || 'master') === 'master')
```

Comment:
Same as above. Address the company by name that requires the exception. Keep the `latest === 'master'` as the else condition.